frontend/drivers/platform_wiiu.c

```diff
@@ -457,53 +463,92 @@ static bool swap_is_pending(void *start_time)
 
 static void init_network(void)
 {
+   ACInitialize();
+   ACConnect();
 #ifdef IS_SALAMANDER
    socket_lib_init();
 #else
    network_init();
 #endif /* IS_SALAMANDER */
 }
 
+static void deinit_network(void)
+{
+   ACClose();
+   ACFinalize();
+}
```

Comment:
are the AC* init/shutdown calls required to happen at global scope ? can't they just be made inside the `getBroadcastAddress()` function ?

Reply:
Not sure. But I expect that doing that will probably disconnect the network and break things like net play and core downloading.

As far as I can tell, we haven’t actually been explicitly connecting to the network—just initializing the network api library and piggy-backing on HBC to have the actual network connection already set up. All this does is make the network connection explicit.

Reply:
ah you mean `ACConnect()` is required for a standalone app ?
ok in that case it can stay.